src/app/home/home.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="jumbotron">
+  <h1 class="display-3">Library Management System</h1>
+  <p class="lead">
+    We created this Library Management System to sharpen and enhance our web
+    development skills as part of the Smoothstack training.
+  </p>
+</div>
+
+<div class="nav-cards">
+  <div class="admin">
+    <div class="card border-primary mb-3" style="max-width: 25rem;">
+      <div class="card-header">LMS</div>
+      <div class="card-body">
+        <a href="#"><h4 class="card-title">Admin</h4></a>
+        <p class="card-text">
+          The admin has access to perform CRUD operations on all of our data
```

Comment:
Technically, the admin doesn't have CRUD abilities on all data entities. Copies and loans are excluded. Though I guess that could change if they were to be included as properties of the entities they're associated with.